hyperopt/tests/test_fmin.py

```diff
@@ -291,3 +291,42 @@ def test_invalid_timeout():
             )
         except Exception as e:
             assert str(e) == expected_message
+
+
+def test_loss_threshold():
+    loss_threshold=0.001
+    hypopt_trials = Trials()
+    best = fmin(
+        fn=lambda x: x ** 2,
+        space=hp.uniform('x', -10, 10),
+        loss_threshold=loss_threshold,
+        algo=rand.suggest,
+        trials=hypopt_trials,
+        return_argmin=False,
+        #max_evals=100
+    )
+    best_loss = hypopt_trials.best_trial['result']['loss']
+    assert best_loss < loss_threshold
+    assert len(hypopt_trials) > 0
+
+
+def test_invalid_loss_threshold():
+    fn = lambda x: [time.sleep(1), x][1]
+    space = hp.choice("x", range(20))
+
+    for wrong_loss_threshold in ["", True]:
+        expected_message = "The loss_threshold argument should be None or a numeric value. Given value: {m}".format(
+            m=wrong_loss_threshold
+        )
```

Comment:
please use black as recommended in the dev guidelines so that the max char length is no more than 88 chars